hls_vi/generate_stac_items.py

```diff
@@ -164,13 +166,13 @@ def add_assets(item, granule, endpoint, version):
     product = item_id.split(".")[1]
     if product == "S30":
         band_info = hls_vi_band_info
-        url = f"https://{endpoint}/lp-prod-protected/HLSS30.{version}/{item_id}/"
-        public_url = f"https://{endpoint}/lp-prod-public/HLSS30.{version}/{item_id}/"
+        url = f"https://{endpoint}/lp-prod-protected/HLS-VIS30.{version}/{item_id}/"
+        public_url = f"https://{endpoint}/lp-prod-public/HLS-VIS30.{version}/{item_id}/"
 
     if product == "L30":
         band_info = hls_vi_band_info
-        url = f"https://{endpoint}/lp-prod-protected/HLSL30.{version}/{item_id}/"
-        public_url = f"https://{endpoint}/lp-prod-public/HLSL30.{version}/{item_id}/"
+        url = f"https://{endpoint}/lp-prod-protected/HLS-VIL30.{version}/{item_id}/"
+        public_url = f"https://{endpoint}/lp-prod-public/HLS-VIL30.{version}/{item_id}/"
```

Comment:
Tests are now broken because you did not _also_ update the fixture file. At this point, don't bother making more changes until we know _exactly_ what the change should be, but when we do know, fix things in this code _and_ in the corresponding fixture file.

Reply:
I updated the fixture. I saw that after pushing the code.